File: Difficulty: Easy/Check if array is sorted/check-if-array-is-sorted.java
class Solution {
    public boolean isSorted(int[] arr) {
    int len = arr.length;
    int num = arr[0];
    boolean result = true;
    for(int i = 1; i < len; i++){
    if( num > arr[i] ){
        result = false;
        
    }    
    else{
        
        num = arr[i];
      
    }
    }
    
    
    return result;
    
    
    
    
    
    
        
    }
}

File: Difficulty: Easy/Check if array is sorted/check-if-array-is-sorted.cpp
class Solution {
  public:
    bool isSorted(vector<int>& arr) {
        // code here
        int length = arr.size();
        for(int i=0;i<length-1;i++){
            if(arr[i]>arr[i+1]){
                return false;
            }
        }
        return true;
    }
};